### DISCUSSION TITLE
how to use "Charge voltage control" available in v0.13?

### DISCUSSION THREAD

**Discussion by hansgeorgk** [Louisvdw/dbus-serialbattery#264] [Q&A] [answered]
Hi @pawcio50501 ,

can you please provide a description on how to enable this new feature ;-)
I tried to use the obvious setting, but then the driver did not start at all. From the list of settings I might need to set multiple values ...

thanks
cheers
HG

**Comment by Louisvdw** [comment, marked as answer]
If you don't use different voltages you should be able to just change the setting in utils.py to enable it.
CVCM_ENABLE = True

**Reply by itenginex** [reply]
Please help me understand what this setting exactly does once enabled. Can you add a feature description? How will the behaviour change? When should I make use of it (Use Case)?

**Reply by Louisvdw** [reply]
It's like having a float stage in your lead acid charging. Once your battery is fully charged, the charge voltage will be lowered. 
I'll do a more detailed writeup for it on the wiki.

**Reply by itenginex** [reply]
Ok, sounds like something I don't want to enable for a LifePo4 battery type

**Comment by hansgeorgk** [comment]
What is the intended use of CVCM - shall I use CVCM together with CCCM?
or shall I better use the one or the other?

**Reply by mr-manuel** [reply]
I would recommend to enable both.

**Reply by mr-manuel** [reply]
https://github.com/Louisvdw/dbus-serialbattery/blob/11676e66b3ae1313b9a58bc778f4d6a97fdd8c12/etc/dbus-serialbattery/config.default.ini

Here is a clear description of all
